packages/client/components/TransactionDetails.tsx

```diff
@@ -0,0 +1,255 @@
+import React from "react";
+import { Text, HStack, Box, VStack, Badge, ScrollView } from "native-base";
+import { FontAwesomeIcon } from "@fortawesome/react-native-fontawesome";
+import { Pressable, StyleSheet } from "react-native";
+import { Transaction } from "@cryptify/common/src/domain/entities/transaction";
+import { faCircleArrowDownLeftCustom } from "./icons/faCircleArrowDownLeftCustom";
+import { faCircleArrowUpRightCustom } from "./icons/faCircleArrowUpRightCustom";
+import { faCopyCustom } from "./icons/faCopyCustom";
+import * as Clipboard from "expo-clipboard";
+
+type Props = {
+    transaction: Transaction;
+    walletAddress: string;
+};
+
+export function TransactionDetails({ transaction, walletAddress }: Props) {
+    const isIncommingTransaction = walletAddress == transaction.walletIn;
+
+    const copyToClipboard = async (valueToCopy: string) => {
+        await Clipboard.setStringAsync(valueToCopy);
+    };
+
+    function formatTransactionAddress(address: string): string {
+        return `${address.substring(0, 6)}...${address.substring(address.length - 4, address.length)}`;
+    }
+
+    function getCurrencyType(): string {
+        return transaction.transactionAddress.substring(0, 2) == "0x" ? "ETH" : "BTC";
+    }
+
+    function renderHeader() {
+        return (
+            <Box style={styles.itemWrapper}>
+                <VStack>
+                    <FontAwesomeIcon
+                        icon={isIncommingTransaction ? faCircleArrowDownLeftCustom : faCircleArrowUpRightCustom}
+                        style={isIncommingTransaction ? styles.receiveIcon : styles.sendIcon}
+                        size={48}
+                    />
+                    <Text
+                        color={isIncommingTransaction ? "success.600" : "text.600"}
+                        style={isIncommingTransaction ? styles.transactionAmountIn : styles.transactionAmountOut}
+                    >
+                        {isIncommingTransaction ? "+" : "-"}
+                        {transaction.amount + " "} {getCurrencyType()}
+                    </Text>
+                    <Text color="text.500" style={styles.transactionsAddress}>
+                        {formatTransactionAddress(transaction.transactionAddress)}
+                    </Text>
+                </VStack>
+            </Box>
+        );
+    }
+    function renderBasicInfo() {
+        // TODO
+        // dynamically get the wallet name if any
+        return (
+            <Box style={styles.itemWrapper}>
+                <VStack>
+                    <Box>
```

Comment:
All sub items with a title and description have a 3px spacing
```suggestion
                    <VStack space="3">
```